### DISCUSSION TITLE
Use remix without backend

### DISCUSSION THREAD

**Discussion by Leastrio** [remix-run/remix#4254] [Q&A]
I absolutely love the styling of remix, but i'd like to know if there was a way to use it with another backend. I've got a backend all in Phoenix and would prefer not to rewrite it all.

**Comment by kiliman** [comment]
You can use React Router v6.4. This has a lot of the features of Remix (like loaders, actions, error boundaries, etc), but is entirely client-side.

https://reactrouter.com/

**Comment by sergiodxa** [comment]
Aside of using React Router, you could fetch your Phoenix-powered backend from Remix loaders and actions, so your backend is Phoenix and Remix works as a [backend for frontend](https://remix.run/docs/en/v1/guides/bff).
